translations/config.json

```diff
@@ -117,7 +119,6 @@
         "LibreSign libresign",
         "mwalbeck nextcloud-breeze-dark",
         "nickv-nextcloud event_update_notification",
```

Comment:
Hello @nickvergessen , looks like you transferred repo "event_update_notification" to "nextcloud/" as well, or?

Reply:
Maybe this should be changed in another commit.